test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -3835,7 +3835,7 @@ def matcher_check_fn():
             include_ops = [
                 "aoti_torch_cpu__weight_int4pack_mm_cpu_tensor"
                 if torch._inductor.config.cpp_wrapper
-                else "extern_kernels.int4mm_packed_weight_cpu"
+                else "torch.ops.quantized.int4mm_packed_weight_cpu.default"
```

Comment:
@Xia-Weiwen , as we synced offline, the test cases do not cover `aoti_torch_cpu__weight_int4pack_mm_cpu_tensor`. Could you help elaborate on the changes? How do the changes test `aoti_torch_cpu__weight_int4pack_mm_cpu_tensor`?

Reply:
It seems that the UT runs both the fallback kernel and the template-based kernel for max-autotune. So, the fallback kernel should be used for codegen, compiling with gcc and benchmarking.